### DISCUSSION TITLE
Not suggestion, but the truth

### DISCUSSION THREAD

**Discussion by Justman100** [Paymenter/Paymenter#418] [Ideas]
So, watch out now... I like to support others, even if it costs me a lot of time and effort! But I do not tolerate being treated like shit in any way! Someone who warns someone twice for the SAME REASON, puts them in timeout, criticises them for not reading the rules (which no one does! At least not right after joining) and bans them for, well, not even a reason, has no open source project or anything to manage or even own!

It's sad how everyone thinks they can treat their users however they want! It's a shame, the software is really good, but it's useless if the owner is an assehole!

In short, I'm gone! That's another way to lose users... Well...

**Comment by MayiVT** [comment]
That's not how it went but ok ❤️ 
Even if you don't know the rules, the rules apply <3

**Comment by PhillipEis** [comment]
Thats not how it went.

You mentioned my PR, but as they said a whole rewrite is going on :-)

My PR is not important, people themself can put the translations in the folder
